package.json

```diff
@@ -1,83 +1,61 @@
+  "module": "dist/react-topology.es.js",
+  "entry": "src/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "start": "styleguidist server",
+    "fmt": "prettier --write --single-quote src/*.js",
+    "lint:js": "eslint . --fix",
+    "lint:css": "stylelint './src/**/*.js'",
+    "test": "run-s lint:js lint:css jest",
+    "jest": "jest --env=jsdom",
+    "build": "NODE_ENV=production bup",
+    "prepublish": "NODE_ENV=production bup",
+    "precommit": "lint-staged"
+  },
   "dependencies": {
-    "babel-plugin-transform-react-remove-prop-types": "^0.4.10",
-    "chart.js": "^2.7.0",
     "d3": "^4.11.0",
-    "force-array": "^3.1.0",
-    "fs": "^0.0.1-security",
-    "joyent-ui-toolkit": "^2.1.0",
     "lodash.difference": "^4.5.0",
     "lodash.differenceby": "^4.8.0",
     "normalized-styled-components": "^1.0.17",
     "polished": "^1.8.1",
-    "prop-types": "^15.5.10",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "react-redux": "^5.0.6",
-    "react-router-dom": "^4.2.2",
-    "react-scripts": "^1.0.14",
-    "reduce-css-calc": "^2.1.1",
-    "redux": "^3.7.2",
-    "redux-form": "^7.1.1",
+    "prop-types": "^15.6.0",
     "remcalc": "^1.0.9",
     "styled-components": "^2.2.1",
     "styled-is": "^1.1.0"
   },
-  "scripts": {
-    "start": "styleguidist server",
-    "build": "styleguidist build",
-    "lint:js": "eslint src",
-    "jest": "jest --env=jsdom",
-    "test": "run-s lint:js lint:css jest",
-    "lint:css": "stylelint './src/**/*.js'",
-    "precommit": "lint-staged"
-  },
   "devDependencies": {
-    "babel-loader": "^7.1.2",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-transform-es3-member-expression-literals": "^6.22.0",
-    "babel-plugin-transform-es3-property-literals": "^6.22.0",
-    "babel-plugin-uglify": "^1.0.2",
-    "babel-preset-env": "^1.6.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-joyent-portal": "^3.2.0",
-    "babel-preset-stage-0": "^6.24.1",
-    "bundlesize": "^0.15.3",
-    "css-loader": "^0.28.7",
-    "enzyme": "^3.1.0",
-    "enzyme-adapter-react-16": "^1.0.1",
-    "enzyme-to-json": "^3.1.2",
-    "eslint": "^4.7.2",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.10",
+    "babel-preset-joyent-portal": "^3.3.3",
+    "bup": "^2.0.6",
+    "eslint": "^4.9.0",
     "eslint-config-joyent-portal": "^3.1.0",
     "eslint-config-prettier": "^2.6.0",
     "eslint-config-react-app": "^2.0.1",
-    "eslint-config-xo-space": "^0.16.0",
+    "eslint-config-xo-space": "^0.17.0",
     "eslint-plugin-flowtype": "^2.39.1",
-    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
+    "eslint-plugin-prettier": "^2.3.1",
     "eslint-plugin-react": "^7.4.0",
-    "file-loader": "^1.1.5",
-    "husky": "^0.14.3",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^21.1.0",
     "joyent-react-scripts": "^2.3.0",
-    "lint-staged": "^4.2.2",
-    "npm-run-all": "^4.1.1",
```

Comment:
You can't remove this , its used for tests

Reply:
you're right 👍